CDP4WebServices.API/Services/Operations/OperationProcessor.cs

```diff
@@ -764,17 +775,26 @@ private DtoResolveHelper GetContainerInfo(Thing thing)
         /// <param name="partition">
         /// The partition.
         /// </param>
+        /// <param name="resolveFromCache">
+        /// A value indicating whether meta data shall be resolved from the CAHCE tables or from the VIEWs
+        /// </param>
         /// <param name="fileStore">
         /// The file Store.
         /// </param>
         private void ApplyOperation(
             CdpPostOperation operation,
             NpgsqlTransaction transaction,
             string partition,
+            bool resolveFromCache,
             Dictionary<string, Stream> fileStore)
         {
             // resolve any meta data from the persitence store
+            if (resolveFromCache)
```

Comment:
This change makes some Integration Tests fail. Book, Page and Section AddAndDelete TestFixtures fail, because somehow the EngineeringModel change is not returned. Seems like the Revision Number is not raised.